### ISSUE TITLE
(AttributeError) "module 'PIL.Image' has no attribute 'VERSION'".

### ISSUE THREAD

**Issue by vineet131** [OpenTTD/nml#39]
I am working on a Windows 64 bit laptop with Conda v2019.03, running a Python 3.6.0 environment. After installing all the dependencies, when I try to run `nmlc -h` it shows me the following error:
`nmlc ERROR: nmlc: An internal error has occurred:
nmlc-version: 0.4.5
Error:    (AttributeError) "module 'PIL.Image' has no attribute 'VERSION'".
Command:  ['C:\\ProgramData\\Anaconda3\\envs\\py3.6\\Scripts\\nmlc']
Location: File "c:\programdata\anaconda3\envs\py3.6\lib\site-packages\nml\version_info.py", line 74, in get_lib_versions`

**Workaround**
After a quick round of googling, I found that the issue is with Pillow v6.0.0. Uninstalling it and doing `pip install pillow==5.4.1` solved the issue for me.

I hope this helps anyone else who might have the same issue.

**Comment by LordAro** [comment]
Thanks for reporting the issue - it had already been noticed (but no issue raised) and pull requested, but not actually merged in or fix released

The first part is now done :)

**Comment by LordAro** [comment]
Reopen as per comment on #29 - should use `PIL.__version__`

**Comment by matthijskooijman** [comment]
Actually, it also seems that `PIL.VERSION` was something different than `PIL.PILLOW_VERSION` and `PIL.__version__`. I suspect the former is the last "PIL" version, frozen when it was forked and the latter two are the "Pillow" version, incremented after the fork. See also https://stackoverflow.com/a/49722022/740048 However, AFAICS this version number is only used for display in `--help` output, so it makes sense to use the Pillow version instead.

I was also wondering since what version `PIL.__version__` is available, so whether a compatibility fallback would be needed (for git master this is not needed, since the minimum pillow version is 5.2, but I'm also looking to backport this fix into the 0.4.5 Debian version). Seems it was first available in 3.4.0: https://github.com/python-pillow/Pillow/commit/7bfb2527bb0b832f830f87c37bd165028118e442, so that's old enough for Debian (oldstable has 4.0.0).

**Comment by ConiKost** [comment]
Fow now, added to use `PIL.__version__`, since it works for me here and documentation states, that `PILLOW_VERSION` will be removed on >=pillow-7.0. I am not sure, if that's the version, we need, but other variables will be removed soon?

```
Python 3.6.9 (default, Jul 15 2019, 00:24:41)
[GCC 9.1.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from PIL import Image
>>> print('PIL',Image.__version__)
PIL 6.2.0
>>>
```
